src/System.Text.Formatting/System/Text/Parsing/InvariantParser_byte.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System.Diagnostics;
+using System.Text.Formatting;
+using System.Text.Utf8;
+
+namespace System.Text.Parsing
+{
+    public static partial class InvariantParser
+    {
+        /// <summary>
+        /// Parses an unsigned 64-bit integer from a location within a UTF-8 byte array buffer.
+        /// </summary>
+        /// <param name="text">The UTF-8 buffer, passed as a byte array.</param>
+        /// <param name="startIndex">The index location of the value to be parsed within the buffer.</param>
+        /// <param name="value">The parsed value.</param>
+        /// <param name="bytesConsumed">The length (in bytes) of the unparsed value within the UTF-8 buffer.</param>
+        /// <returns>True if parsing is successful; false otherwise.</returns>
+		public static bool TryParse(byte[] text, int startIndex, out byte value, out int bytesConsumed)
```

Comment:
Feel free to fix it in a subsequent commit, but I think the parameters should be renamed:
1. The first one should be called utf8Text
2. startIndex should be called simply "index". This is a convention we use in many other places in the Framework.